### PR TITLE
feat(tmux): Installing 'tmux-resurrect' and 'tpm' plugin to resurrect the previous saved session in tmux.

### PR DESCRIPTION
**Pull request by harshv5094**
I've modified the code and deleted the dashboard plugin that I've added to the setup. Now It only has tmux resurrect and tpm plugin

### CODE REVIEW

File: .config/nvim/lua/craftzdog/plugins.lua [hunk 1]
```diff
@@ -49,5 +49,6 @@ packer.startup(function(use)
   -- use 'github/copilot.vim'
 
   use 'lewis6991/gitsigns.nvim'
-  use 'dinhhuy258/git.nvim' -- For git blame & browse
+  use 'dinhhuy258/git.nvim'    -- For git blame & browse
+  use 'glepnir/dashboard.nvim' -- LSP UIs
```
[the remark this answers is not in the document]
Comment (harshv1741) [reply .config/nvim/lua/craftzdog/plugins.lua]: Thanks for the suggestion

**Comment by harshv1741** [comment]
I've installed tmux plugin manager to install tmux-resurrect which allows to save and reload sessions, even after rebooting the system.

To use the plugins, first clone tpm:
git clone https://github.com/tmux-plugins/tpm ~/.tmux/plugins/tpm

After clonning just copy paste my changes in tmux config and open tmux.

press prefix(In craftzdog setup Ctrl-T) + I to install plugins I've written in .config/tmux/plugins.conf

the tpm will automatically install all the plugin.

to save the session : Ctrl - T Ctrl - s
to restore the session: Ctrl - T Ctrl - r

**Comment by danilosh45** [comment]
Hello, i know my question is no related to this Issue, but i want to know what is the leader key definition is the takuya configuration. :)

**Comment by mPanasiewicz** [comment]
> Hello, i know my question is no related to this Issue, but i want to know what is the leader key definition is the takuya configuration. :)

`\`

**Pull request closed without merging**